### ISSUE TITLE
duplicated results in search in TYPO3 BE

### ISSUE THREAD

**Issue by tmotyl** [einpraegsam/femanager#6]
In "frontend users" module in TYPO3 backend if you search for a user
you'll get multiple results for the same user.

How to reproduce:
1) assign user to multiple usergroups
2) search for him
3) you will get as many results as he has usergroups assigned

Additionally the search with many users (130k) is quite slow. Is it possible to configure fields which are taken into account for searching?

**Comment by einpraegsam** [comment]
I can reproduce this (old) issue in current develop branch. That is caused of 
```
$or[] = $query->like('usergroup.title', '%' . $searchword . '%');
```
Possibilities (BTW: Extbase doesn't support a "group by" setting):
* Removing this line which will result in not searching by usergroup title any more
* Filtering the result of unique values in an array. But this conflicted with the pagebrowser because it needs a queryresult
* Doing an own SQL query for the repository method

Long story short: I will tolerance this issue for now.
